flow/connectors/postgres/postgres.go

```diff
@@ -180,6 +182,9 @@ func (c *PostgresConnector) MaybeStartReplication(
 		defer c.replLock.Unlock()
 		if err := pglogrepl.StartReplication(ctx, c.replConn.PgConn(), slotName, startLSN, replicationOpts); err != nil {
 			c.logger.Error("error starting replication", slog.Any("error", err))
+			if strings.Contains(err.Error(), "is active for PID") {
```

Comment:
check with pgerrors.Is(err, pgerrors.ObjectInUse)

But wondering if we can preserve original error & use pgerrors.Is to check at caller. Need to use errors.As to unwrap error into `*pgconn.PgError`